portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/moresites.vm

```diff
@@ -104,38 +107,38 @@
             <div class="moresites-left-col">
               #foreach( $termKey in $tabsSites.tabsMoreSortedTermList )
                 #if ($tabsSites.tabsMoreTermsLeftPane.get($termKey).size() > 0)
-                  <div class="fav-sites-term">
+                  <div class="fav-sites-term fav-sites-card card mb-2">
+                    <div class="card-header">
                     #if ( !$termKey || $termKey == "" )
-                      <h3 class="favorites-term-header">
-                        <span class="favorites-select-container">
-                          <button class="favorites-select-all-none icon-button"
-                              title="${rloader.moresite_toggle_all_as_favorites_no_term}"
-                              aria-label="${rloader.moresite_toggle_all_as_favorites_no_term}">
-                            <i class="bi-pin-fill" style="display: none"></i>
-                            <i class="bi-pin" style="display: none"></i>
-                          </button>
-                        </span>
+                      <h3 class="card-title">
                         <span>${rloader.sit_notermkey}</span>
+                        <button class="favorites-select-all-none icon-button ms-1"
+                            title="${rloader.moresite_toggle_all_as_favorites_no_term}"
+                            aria-label="${rloader.moresite_toggle_all_as_favorites_no_term}">
+                          <i class="si si-pin-fill" style="display: none"></i>
+                          <i class="si si-pin" style="display: none"></i>
```

Comment:
```suggestion
                          <i class="si si-pin-fill d-none"></i>
                          <i class="si si-pin d-none"></i>
```

Reply:
I don't want to do that. I've tested the JS to death already. Maybe later.